src/server/api/models/User.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  Model,
+  Table,
+  Column,
+  DataType,
+  AutoIncrement,
+  PrimaryKey,
+  AllowNull,
+  Default,
+  ForeignKey,
+} from 'sequelize-typescript';
+import { Nullable } from '../../../client/types';
+import Theme from './Theme';
+
+type UserAttributes = {
+  id: number;
+  first_name: string;
+  second_name: string;
+  display_name: Nullable<string>;
+  login: string;
+  email: string;
+  phone: string;
+  avatar: Nullable<string>;
+  theme: number;
+};
+
+@Table
+class User extends Model<UserAttributes> {
+  @AutoIncrement
+  @PrimaryKey
+  @Column(DataType.INTEGER)
+  id!: number;
+
+  @AllowNull(false)
+  @Column(DataType.STRING)
+  first_name!: string;
+
+  @AllowNull(false)
+  @Column(DataType.STRING)
+  second_name!: string;
+
+  @Column(DataType.STRING)
+  display_name!: string;
+
+  @AllowNull(false)
+  @Column(DataType.STRING)
+  login!: string;
+
+  @AllowNull(false)
+  @Column(DataType.STRING)
+  email!: string;
+
+  @AllowNull(false)
+  @Column(DataType.STRING)
+  phone!: string;
+
+  @Column(DataType.STRING)
+  avatar!: string;
+
+  @ForeignKey(() => Theme)
+  @AllowNull(false)
+  @Column({
+    type: DataType.INTEGER,
+    field: 'id',
+  })
+  @Default(1)
+  theme!: number;
```

Comment:
@Column field: 'id', это типа так указывается поле для foreign key? просто чет в доке такого не нашел. И не сильно очевидно выглядит.

Логичнее и правильнее назвать поле themeId и сделать
```suggestion
  @ForeignKey(() => Theme)
  @AllowNull(false)
  @Column(DataType.INTEGER)
  @Default(1)
  themeId!: number;
```

А еще очень похоже, что DataType.INTEGER можно не писать для number полей - он типа сам умеет вычленять.

Reply:
Круто! Не знал что сам умеет. Тогда удалю лишние указания `DataType` для `INTEGER` колонок.

А по поводу 
```
@Column({
  ...,
  field: id
})
```

делал по примеру в теории ([связи](https://praktikum.yandex.ru/learn/middle-frontend/courses/ce1c09fb-f189-4b3d-9662-9dde4d271257/sprints/5151/topics/1eb16d47-5fe3-4b2a-b2ce-e8f1a905d897/lessons/27beb206-bcbf-493e-8591-dbaae2a9f39b/)):
```
class Message extends Model<Message> {
    @ForeignKey(() => User)
    @Column({
        type: DataType.INTEGER,
        field: 'owner_id'
    })
    ownerId: number;
}
```